Documentation/gettingstarted/kind.rst

```diff
@@ -0,0 +1,244 @@
+
+With Kind we can simulate Cluster Mesh in a sandbox too.
+
+Kind Configuration
+------------------
+
+This time we need to create (2) ``config.yaml``, one for each kubernetes cluster.
+We will explicitly configure their ``pod-network-cidr`` and ``service-cidr`` to not overlap.
+
+Example ``kind-cluster1.yaml``:
+
+.. code:: yaml
+
+    kind: Cluster
+    apiVersion: kind.x-k8s.io/v1alpha4
+    nodes:
+    - role: control-plane
+    - role: worker
+    - role: worker
+    - role: worker
+    networking:
+      disableDefaultCNI: true
+      podSubnet: 10.0.0.0/16
+      serviceSubnet: 10.1.0.0/16
+
+Example ``kind-cluster2.yaml``:
+
+.. code:: yaml
+
+    kind: Cluster
+    apiVersion: kind.x-k8s.io/v1alpha4
+    nodes:
+    - role: control-plane
+    - role: worker
+    - role: worker
+    - role: worker
+    networking:
+      disableDefaultCNI: true
+      podSubnet: 10.2.0.0/16
+      serviceSubnet: 10.3.0.0/16
+
+Create Kind Clusters
+--------------------
+
+We can now create the respective clusters:
+
+.. code:: bash
+
+    kind create cluster --name=cluster1 --config=kind-cluster1.yaml
+    kind create cluster --name=cluster2 --config=kind-cluster2.yaml
+
+Deploy Cilium
+-------------
+
+This is the same helm command as from :ref:`kind_install_cilium`. However
+we're enabling managed etcd and setting both ``cluster-name`` and
+``cluster-id`` for each cluster.
+
+Make sure context is set to ``kind-cluster2`` cluster.
+
+.. code:: bash
+
+   kubectl config use-context kind-cluster2
+
+.. parsed-literal::
+
+   helm install cilium |CHART_RELEASE| \\
+      --namespace kube-system \\
+      --set global.nodeinit.enabled=true \\
+      --set global.kubeProxyReplacement=partial \\
+      --set global.hostServices.enabled=false \\
+      --set global.externalIPs.enabled=true \\
+      --set global.nodePort.enabled=true \\
+      --set global.hostPort.enabled=true \\
+      --set global.etcd.enabled=true \\
+      --set global.etcd.managed=true \\
+      --set global.identityAllocationMode=kvstore \\
+      --set global.cluster.name=cluster2 \\
+      --set global.cluster.id=2
+
+Change the kubectl context to ``kind-cluster1`` cluster:
+
+.. code:: bash
+
+   kubectl config use-context kind-cluster1
+
+.. parsed-literal::
+
+   helm install cilium |CHART_RELEASE| \\
+      --namespace kube-system \\
+      --set global.nodeinit.enabled=true \\
+      --set global.kubeProxyReplacement=partial \\
+      --set global.hostServices.enabled=false \\
+      --set global.externalIPs.enabled=true \\
+      --set global.nodePort.enabled=true \\
+      --set global.hostPort.enabled=true \\
+      --set global.etcd.enabled=true \\
+      --set global.etcd.managed=true \\
+      --set global.identityAllocationMode=kvstore \\
+      --set global.cluster.name=cluster1 \\
+      --set global.cluster.id=1
+
+Setting up Cluster Mesh
+------------------------
+
+We can continue to follow :ref:`gs_clustermesh` with exposing
+the Cilium etcd quorums using NodePort.
```

Comment:
I think that it would be useful to link directly to the right part of the page:

gettingstarted/clustermesh.html#expose-the-cilium-etcd-to-other-clusters

(presumably using the same technique as for the `ts_agent_logs` link).

Reply:
Done and added info about namespace.